apps/annotation/opencv_annotation.cpp

```diff
@@ -107,42 +110,33 @@ void on_mouse(int event, int x, int y, int , void * )
     }
 }
 
-// FUNCTION : snippet to convert an integer value to a string using a clean function
-// instead of creating a stringstream each time inside the main code
-string int2string(int num)
-{
-    stringstream temp_stream;
-    temp_stream << num;
-    return temp_stream.str();
-}
-
 // FUNCTION : given an image containing positive object instances, add all the object
 // annotations to a known stringstream
-void get_annotations(Mat input_image, stringstream* output_stream)
+void get_annotations(Mat input_image, vector<Rect>* current_annotations)
```

Comment:
Function description became obsolete.
Also, I think it could be more readable if the function returned `vector<Rect>`:
```.cpp
vector<Rect> get_annotations(Mat input_image) {
    vector<Rect> result;
   ...
   return result;
}
...
annotations.push_back(get_annotations(image));
```